src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -26,21 +26,65 @@ export interface ChannelItemProps
   unread?: boolean;
   /** Content of the badge signaling unread messages, such as a badge displaying the number. */
   unreadBadgeLabel?: string;
+  /** Boolean to indicate if we want to display the channel with last channel message and timestamp */
+  displayLastChannelMessageAndTimestamp?: boolean;
```

Comment:
instead of having this extra prop, maybe you can just check if lastChannelMessage or lastChannelMessageTimestamp has some value or not

Reply:
They should be able to enable timestamp OR preview message, right? This makes it seem like they need to always enable both at the same time